File: Python/01.Python-Basic-April-2019/10.for_loop_lab/08.odd_even_sum.py
n = int(input())

even_sum = 0
odd_sum = 0

for i in range(1, n + 1):
    num = int(input())
    if i % 2 == 0:
        even_sum += num
    else:
        odd_sum += num

diff = abs(even_sum - odd_sum)
if even_sum == odd_sum:
    print("Yes")
    print(f"Sum = {even_sum}")
else:
    print("No")
    print(f"Diff = {diff}")

File: Go/01.Go-Basic-February-2022/07.forLoop-Lab/08.oddEvenSum.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Scanln(&n)

	var sumEven int = 0
	var sumOdd int = 0

	for i := 1; i <= n; i++ {
		var value int
		fmt.Scanln(&value)

		if i%2 == 0 {
			sumEven += value
		} else {
			sumOdd += value
		}
	}

	if sumEven == sumOdd {
		fmt.Println("Yes")
		fmt.Printf("Sum = %d", sumEven)
	} else {
		fmt.Println("No")
		var diff int = sumEven - sumOdd
		fmt.Printf("Diff = %.0f", math.Abs(float64(diff)))
	}
}
